lgc/patch/PatchInOutImportExport.cpp

```diff
@@ -101,6 +100,14 @@ bool PatchInOutImportExport::runOnModule(Module &module) {
   m_hasTs = (stageMask & (shaderStageToMask(ShaderStageTessControl) | shaderStageToMask(ShaderStageTessEval))) != 0;
   m_hasGs = (stageMask & shaderStageToMask(ShaderStageGeometry)) != 0;
 
+  std::vector<Function *> candidateCallees;
```

Comment:
nit: SmallVector?

Reply:
A question has confused me for some time. Do we have explicit preference for llvm-data-structures over stl-data-structures? I have seen both used widely inside llpc. Is there any previous discussion on this? If we have reached an agreement, it would be easy for us to follow.

Reply:
When it comes to hash tables, llvm ones are generally faster than the stl ones in my experience. SmallVectors make sense when you expect the number of elements to be small and can avoid dynamic allocations. E.g., one recent commit where it made a difference: https://reviews.llvm.org/D87319. 
I think this seems to be the case here, doesn't it?

Reply:
make sense, already fixed locally. Thanks!